src/saftvrmie/eos/mod.rs

```diff
@@ -0,0 +1,113 @@
+use crate::hard_sphere::HardSphere;
+
+use super::SaftVRMieParameters;
+use association::Association;
+use feos_core::{
```

Comment:
not a major issue, but I feel like this default setting of how rust-analyzer does imports is one of the weaker decisions they made. I know you can configure it, but is there a way to do it workspace-specific and include the config file in GitHub?